vagrant/provisioning/roles/arkcase-prerequisites/tasks/main.yml

```diff
@@ -1,3 +1,13 @@
+- name: set Java 8 as Java Home (old versions versions)
+  set_fact:
+    java_home_version: "/usr/lib/jvm/java-1.8.0-openjdk"
+  when: arkcase_version != "" and arkcase_version is version('2023.02', '<')
+
+- name: set Java 11 as Java Home (new versions versions)
+  set_fact:
+    java_home_version: "/usr/lib/jvm/java-11-openjdk"
+  when: arkcase_version == "" and arkcase_version is version('2023.02', '>=')
```

Comment:
you need `or` here, since the version won't be both blank and greater. than 2023.02.  We want Java 11 if the version is blank, or 2023.02+.

Reply:
Fixed